src/notebook.jl

```diff
@@ -173,6 +173,35 @@ macro ingredients(filename)
 	end
 end
 
+# ╔═╡ 36a67c48-f398-4ec1-9f01-b33d647137ee
+"""
+Run a code block using debounce, i.e. only run it when variable list has stabilized through `timeout`.
+
+```julia
+@use_debounce([var], 1) do
+	sleep(2.)
+	@info "stable variable \$var"
+end
+```
+"""
+macro use_debounce(f, deps, timeout)
+	quote
+		ref, set_ref = @use_state(nothing)
+		count_ref = @use_ref(0)
+
+		count_ref.x += 1
```

Comment:
I think `ref[]` is the prefered way of interacting with a `RefValue` (i did not know about `ref.x`).
```suggestion
		count_ref[] += 1
```

Reply:
I didn't know about `ref[]` 😆